taglib/fileref.cpp

```diff
@@ -181,6 +182,9 @@ namespace
       return new XM::File(arg, readAudioProperties, audioPropertiesStyle);
     if (ext == "DSF")
       return new DSF::File(arg, readAudioProperties, audioPropertiesStyle);
+    if (ext == "MKA" || ext == "MKV") {
```

Comment:
TagLib style is without spaces between function and opening parenthesis.

If you would, Please also fix the DSF one that was previously missed right above as well ... Thank you!